pkg/ebpf/tracee.go

```diff
@@ -1639,3 +1643,62 @@ func (t *Tracee) triggerSeqOpsIntegrityCheckCall(
 	seqOpsStruct [len(derive.NetSeqOps)]uint64) error {
 	return nil
 }
+
+// triggerMemDump is used by a Uprobe to trigger an eBPF program
+// that prints the seq ops pointers
+func (t *Tracee) triggerMemDump() error {
+	_, ok := t.events[events.PrintMemDump]
+	if !ok {
```

Comment:
nit - you can do instead
`if _, ok := t.events[events.PrintMemDump]; !ok {`
which is shorter. This is used when the return variable (ok in this case) is not required after the check.

I saw this pattern repeated in several places in this PR, so if you want you can fix as well